docs/hosting/environment-variables/environment-variables.md

```diff
@@ -118,7 +117,7 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
 | `N8N_SMTP_SSL` | Boolean | `true` | Whether to use SSL for SMTP (true) or not (false). |
 | `N8N_UM_EMAIL_TEMPLATES_INVITE` | String | - | Full path to your HTML email template. This overrides the default template for invite emails. |
 | `N8N_UM_EMAIL_TEMPLATES_PWRESET` | String | - | Full path to your HTML email template. This overrides the default template for password reset emails. |
-| `N8N_USER_MANAGEMENT_JWT_SECRET` | String | - | Set a specific JWT secret. By default, n8n generates one on start. |
```

Comment:
this is still relevant, as it's used by the JWTs that we generate. We are only removing the **external** JWT support.

Reply:
Fixed, thanks